src/controllers/connectionManager.ts

```diff
@@ -722,6 +735,23 @@ export default class ConnectionManager {
         });
     }
 
+    /**
+	 * Gets the connection string to send to the middleware
```

Comment:
>	  [](start = 0, length = 2)

nit: spacing (you can visualize this in VS Code with View: Toggle Render Whitespace command to make this easy to spot)